ProcessMaker/Http/Controllers/TaskController.php

```diff
@@ -148,8 +149,11 @@ public function edit(ProcessRequestToken $task, string $preview = '')
             ]);
             $userConfiguration = (new UserConfigurationController())->index();
 
+            $taskResource = new TaskEditResource($task);
+            $taskJson = $taskResource->toArray(request());
+
             return view('tasks.edit', [
-                'task' => $task,
+                'task' => json_decode(json_encode($taskJson)),
```

Comment:
Consider to add an additional variable to be used in the `const task = @json($taskJson);`:
```suggestion
                'task' => $task,
                'taskJson' => json_decode(json_encode($taskJson)),
```